superset/mcp_service/sql_lab/schemas.py

```diff
@@ -41,8 +45,12 @@ class ExecuteSqlRequest(BaseModel):
     timeout: int = Field(
         default=30, description="Query timeout in seconds", ge=1, le=300
     )
-    parameters: dict[str, Any] | None = Field(
-        None, description="Parameters for query substitution"
+    template_params: dict[str, Any] | None = Field(
```

Comment:
I replaced the `{ parameter }` syntax with Jinja `{{ parameter }}` as I'm assuming that would be ok. Let me know if you see a problem with that.

Reply:
@aminghadersohi The MCP is not part of an official release yet and `UPDATING.md` is reserved for when you are upgrading between versions. By the way, we should remove [this documentation](https://github.com/apache/superset/blob/ee45b26ad7a9233fd747864b3e589c6d7144eaac/UPDATING.md?plain=1#L27) from `UPDATING.md`. This is more appropriate for Release Notes or a page in the Developer Portal. @aminghadersohi Could you open a PR to remove those docs from `UPDATING.md`?

Reply:
cc @sadpandajoe as the release manager.